### DISCUSSION TITLE
有鸿蒙系统的app没

### DISCUSSION THREAD

**Discussion by amdfanvip** [localsend/localsend#1799] [L-简体中文]
有鸿蒙系统的app没，能出个鸿蒙版本的吗

**Comment by 1-hexene** [comment]
自己编译去

**Comment by amdfanvip** [comment]
要会就不问了😂qq在 2024年9月23日，13:34，王淑宁 ***@***.***> 写道：﻿
自己编译去

**Comment by Timothy-tian** [comment]
希望能有大佬写个鸿蒙NEXT的版本

**Comment by azhu003** [comment]
https://github.com/azhu003/localsend-harmony.git

**Comment by lawrenceching** [comment]
测了一下，鸿蒙NEXT下用卓易通安装 LocalSend ，功能正常。

**Comment by xiaoxing365** [comment]
nextmax显示鸿蒙市场上架了一个localsend，也不知道能用不

**Reply by niuguihua** [reply]
鸿蒙市场的localsend好像是个李鬼

**Comment by 4SM** [comment]
既然用上了鸿蒙，为什么不选择爱国的软件？

**Comment by cmlanche** [comment]
闪电藤试试

**Reply by intfish123** [reply]
闪电藤好用，卓易通的localsend用不了
